File: Python/Basic1/Exercise010.py
# Write a program that accepts an integer (n) and computes the value of n+nn+nnn.

num = input("Give me a number: ")

n1 = int("%s" %(num))
n2 = int("%s%s" %(num, num))
n3 = int("%s%s%s" %(num, num, num))

print(n1 + n2 + n3)

File: C/Basic Declarations/Exercise010.c
#include <stdio.h>

//Write a program that accepts two integers from the user and calculates the product of the two integers.

int main() {

	int first, second;

	printf("Give me a integer: ");
	scanf_s("%d", &first);

	printf("Give me another integer: ");
	scanf_s("%d", &second);

	printf("\nThe product of both numbers is %d\n", first * second);

	return 0;
}
